server/src/data.py

```diff
@@ -496,6 +498,22 @@ def mouseClicked(self, event, x, y, flags, extraParameter):
                 self.interestList.remove(toRemove)
         # Redraw self.video on vWindow.
         cv2.imshow(self.vWindow, self.video)
+	
+    def quit(self):
+        cv2.destroyAllWindows()
+        self.videoCapture.release()
+        sys.exit()
+
+    def checkWindowsStatus(self):
+        """
+        If any is closed, close all.
+        """
+        for window in self.windows:
+            if window != self.hWindow:
+                windowStatus = cv2.getWindowProperty(window, 0)
+                if windowStatus < 0:
```

Comment:
Can you add a comment about why this works? (Eg, that `getWindowProperty` returns -1 if the window is closed?)